backend/src/models/event.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose')
+const { AttendeeSchema } = require('./attendee')
+
+const Event = new mongoose.Schema({
+  name: { type: String },
+  date: { type: String },
+  startTime: { type: Number },
+  endTime: { type: Number },
+  location: { type: String },
+  description: { type: String },
+  attendees: { type: [AttendeeSchema] } // subdocument
```

Comment:
make these attributes all required, since you're requiring that every field is present in your endpoint